client/my-sites/stats/pages/subscribers/index.tsx

```diff
@@ -89,7 +92,7 @@ const StatsSubscribersPage = ( { period }: StatsSubscribersPageProps ) => {
 				{ isLoading && <StatsModulePlaceholder className="is-subscriber-page" isLoading /> }
 				{ ! isLoading &&
 					( showLaunchpad ? (
-						<SubscriberLaunchpad launchpadContext="subscriber-stats" />
```

Comment:
Is it alright that we skip setting`launchpadContext`?

Reply:
Fixed in [8d95ef0](https://github.com/Automattic/wp-calypso/pull/87657/commits/8d95ef0c19b4bda40f4c575ee54c49d8e69639aa).